app.test.js

```diff
@@ -0,0 +1,112 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/test', {useMongoClient: true});
+mongoose.Promise = global.Promise;
+
+const Team = require('./TeamModel');
+const server = require('./server')
+
+const chai = require('chai');
+const assert = chai.assert;
+const chaiHTTP = require('chai-http');
+
+chai.use(chaiHTTP);
+describe('App Routes', () => {
+  describe('/allteams', () => {
+    beforeEach((done) => {
+      Team.remove({}, (err) => {
+        if (err) return console.log(err);
+        done();
+      })
+    });
+    describe('[GET] /allteam', () => {
+      it('should return all the teams in the database', (done) => {
+        chai.request(server)
+          .get('/api/allteams')
```

Comment:
I think in this test, you would want to insert a few entries into the db first and then perform the GET, so that you're actually testing that it is working. Just performing a GET on an empty db isn't a good test for this functionality.